src/qt/editor/toolsmenu.cpp

```diff
@@ -181,6 +183,78 @@ ToolsMenu::ToolsMenu(InviwoMainWindow *win) : QMenu(tr("&Tools"), win) {
     auto sourceMenu = addMenu("Create &Sources");
     addInviwoMetaAction(sourceMenu);
 #endif
-}
+
+    auto workspaceMenu = addMenu("Workspaces");
+
+    auto loadExampleWorkspaces = workspaceMenu->addAction("Load Example Workspaces");
```

Comment:
could add a tooltip to those actions explaining the difference between load (dryrun::yes) and update (dryrun::no)

Reply:
mmm